contents/blog/best-open-source-session-replay-tools.md

```diff
@@ -0,0 +1,195 @@
+### Is it open source?
+
+No. UXlens is not open source.
+
+## 5. [SessionStack](https://www.sessionstack.com/)
+
+![SessionStack - open source GA alternative](../images/blog/open-source-hotjar-alternatives/img6_SessionStack.png)
+
+SessionStack is a user session insights tool with a focus on critical issues and error tracing. Without additional analytics features, it doesn’t necessarily give a cohesive overview of the user experience at large or assist with product development. Its tooling is particularly effective for customer support use cases; error messages and co-browsing sessions help teams to pinpoint issues and track down problems related to specific users, devices or releases.
+
+### Who is SessionStack for?
+
+SessionStack enables product managers, customer support teams and engineers to harness user navigation insights to identify friction points and bugs, and to assist users in real-time.
+
+### Strengths
+
+- Notification alerts when issues arise
+- Network traffic monitoring
+- Ability to co-browse with users to offer guidance
+- Cloud-hosting and self-hosting options available
+
+### How much does it cost?
+Cloud plans with SessionStack start from $99 a month for up to 1,000 sessions. A self-hosted deployment option is available on custom plan, which includes implementation support and customisable data volume and retention. Pricing for SessionStack’s self-hosted plan is not publicly available.
+
+### Is it open source?
+
+SessionStack’s code is not open source. It does offer a self-hosted deployment option, though, which allows customers to keep their user data off the cloud, but you can expect to pay a premium for the privilege.
+
+## 6. [Matomo](https://matomo.org/)
+![Matomo - open source GA alternative](../images/blog/open-source-hotjar-alternatives/Matomo-session-recording.png)
+
+Matomo is an open web analytics platform. Touted as an alternative to Google Analytics, Matomo is largely focused on understanding marketing website analytics. Session recording is available as an add-on via the [on-premise marketplace](https://plugins.matomo.org/HeatmapSessionRecording).
+
+### Who is Matomo for?
+
+Matomo is built with marketing and content teams in mind, offering insights into website content engagement for optimization of user journeys.
+
+### Strengths
+
+- Funnels, acquisition analysis and other insight features
+- Unlimited session recordings
+- Out-of-the-box solution meaning no developer required to get set up
+
+### How much does it cost?
+
+The session recording add-on for a self-hosted Matomo solution has a 30-day free trial, after which the cost ranges from $199 to $599 per year depending on the number of users in your team.
+
+Session recording is also included in Matomo’s cloud Business plan. Pricing is calculated depending on the volume of your site traffic. 
+
+### Is it open source?
+
+Yes. Matomo is available under a AGPLv3 license. Check out [the Matomo repo](https://github.com/matomo-org/matomo) for more info.
```

Comment:
Would be good to have some sort of conclusion here, in my opinion. A recap of why PostHog is best, for who.

Reply:
Yes to a CTA, but I don't think we should get too much into "why we're better than x". Save that for comparison pages.

Reply:
Will leave it to you, but as this page is already ranking and listing competitors anyway I don't think that's a reason to default to putting the content elsewhere (esp. when we don't have comparison pages yet and probably need to re-align that project). 

We don't need to have a comparison chart or feature discussion here, but some sort of conclusion to the article based on what's already on the page makes sense to me.